docs/AMF/using-amf/parsing.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: amf_parsing
+title: Parsing with AMF
+---
+
+AMF is capable of parsing things like:
+
+- AML dialects
+- API specifications
+  - RAML
+  - OAS 2 & 3
+  - AsyncAPI 2.0
+
+In this section you'll learn how to parse each one of them, followed by practical examples.
+
+
+## Parsing Basics
+Let's cover what a parser does. Parsing is the process of analyzing text made of a sequence of tokens, and then building a data structure based on those tokens.
+
+AMF uses [SYAML](https://github.com/aml-org/syaml) to read JSON or YAML files and generate [ASTs](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (abstract syntax tree)
```

Comment:
True, I've capitalized them because we're used to doing it that way, I'll ask @looseale if they can be lowercase or should always be capitalized (He's doing the AML documentation).

Reply:
I've changed all instances of 'AML Dialect' and 'AML Vocabulary' to lowercase.